crm_claim/README.rst

```diff
@@ -0,0 +1,110 @@
+=================
+Claims Management
```

Comment:
module `crm_claim` is already merged in 14.0 (https://github.com/OCA/crm/pull/391)
can you please remove from this PR?